t/torchvision/torchvision_ubi_9.3.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : vision
+# Version           : v0.16.1
+# Source repo       : https://github.com/pytorch/vision.git
+# Tested on         : UBI:9.3
+# Language          : Python
+# Travis-Check      : True
+# Script License    : Apache License, Version 2.0
+# Maintainer        : Md. Shafi Hussain <Md.Shafi.Hussain@ibm.com>
+#
+# Disclaimer        : This script has been tested in root mode on given
+# ==========          platform using the mentioned version of the package.
+#                     It may not work as expected with newer versions of the
+#                     package and/or distribution. In such case, please
+#                     contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=vision
+PACKAGE_VERSION=${1:-v0.16.1}
+PACKAGE_URL=https://github.com/pytorch/vision.git
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+PYTHON_VER=${2:-3.9}
+PARALLEL=${PARALLEL:-$(nproc)}
+
+WORKDIR=$(pwd)
+
+dnf install -y https://mirror.stream.centos.org/9-stream/BaseOS/ppc64le/os/Packages/centos-gpg-keys-9.0-24.el9.noarch.rpm \
+            https://mirror.stream.centos.org/9-stream/BaseOS/`arch`/os/Packages/centos-stream-repos-9.0-24.el9.noarch.rpm \
+            https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+dnf config-manager --add-repo https://mirror.stream.centos.org/9-stream/AppStream/`arch`/os
+dnf config-manager --set-enabled crb
+dnf install -y git cmake ninja-build g++ rust cargo jq \
+            python${PYTHON_VER}-devel python${PYTHON_VER}-wheel python${PYTHON_VER}-pip python${PYTHON_VER}-setuptools \
```

Comment:
Remove python installation instructions

Reply:
I do need to install python3-devel, transitive packages require python header files which isn't there in the base image installation of python